internal/pkg/db/redis/models/interval_action.go

```diff
@@ -51,12 +50,11 @@ func (ia IntervalAction) Add() (cmds []DbCommand) {
 			cmds = append(cmds, DbCommand{Command: "SADD", Hash: key + ":" + ia.Target, Key: ia.ID})
 		}
 	}
-	return cmds
+	return
 }
 
 func (ia IntervalAction) Remove() (cmds []DbCommand) {
-	cmds = make([]DbCommand, len(intervalActionKeys))
```

Comment:
Why did you remove the slice initialization (it's explicitly initializing the slice for the proper size)?

Reply:
If length of intervalActionKeys is 3, after slice initialization  there are 3 empty DbCommand structs. According to the below code, it will append more DbCommand struct  which will not replace the empty struct in the slice.
<img width="721" alt="111WX20191231-134922@2x" src="https://user-images.githubusercontent.com/1459780/71611504-65405f00-2bd4-11ea-9c13-bb7f80af47bc.png">

Reply:
You are correct.  However, I wonder if a more efficient solution would be to initialize cmds using make once (since we know its required size in advance) and subsequently assign its individual elements directly (as opposed to using append).